vm/Cargo.toml

```diff
@@ -89,6 +91,8 @@ subprocess = "0.2.2"
 num_cpus = "1"
 socket2 = { version = "0.3", features = ["unix"] }
 rustyline = "6.0"
+openssl = "0.10"
```

Comment:
Have we considered something pure rust? Maybe [RusTLS](https://github.com/ctz/rustls)?

Reply:
I looked into it, but `_ssl`'s API pretty directly corresponds to OpenSSL's API; I don't think it would be guaranteed that everything we needed would be present in rustls.